src/components/modal/index.tsx

```diff
@@ -24,8 +24,8 @@ export interface Props
   title?: string;
   primaryActionButton?: ActionButton;
   secondaryActionButton?: ActionButton;
-  variant?: 'fullScreen' | 'base' | 'topScroll';
-  size?: 'md' | 'lg' | 'full';
+  variant?: 'fullScreen' | 'base' | 'topScroll' | 'grayOut';
```

Comment:
I am not sure variant suits best for this. What if I want a base modal with a gray overlay? 🤔 shouldn't we just pass it as a `overlayColor` prop?

Reply:
Thanks, will take it into account still waiting for the final decisions from the design side and will ask them about their take (it might be particular use case for the login that's why I initially went with a separate variant over exposing another prop to allow for more combinations)

Reply:
Switched it to an optional prop